src/libs/actions/ExitSurvey.ts

```diff
@@ -0,0 +1,42 @@
+import Onyx from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import * as API from '@libs/API';
+import Log from '@libs/Log';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+let exitReason: ValueOf<typeof CONST.EXIT_SURVEY.REASONS> | undefined;
+let exitSurveyResponse: string | undefined;
+Onyx.connect({
+    key: ONYXKEYS.FORMS.EXIT_SURVEY_REASON_FORM,
+    callback: (value) => (exitReason = value?.[CONST.EXIT_SURVEY.REASON_INPUT_ID]),
+});
+Onyx.connect({
+    key: ONYXKEYS.FORMS.EXIT_SURVEY_RESPONSE_FORM,
+    callback: (value) => (exitSurveyResponse = value?.[CONST.EXIT_SURVEY.RESPONSE_INPUT_ID]),
+});
+
+function saveExitReason(reason: ValueOf<typeof CONST.EXIT_SURVEY.REASONS>) {
+    Onyx.merge(ONYXKEYS.FORMS.EXIT_SURVEY_REASON_FORM, {[CONST.EXIT_SURVEY.REASON_INPUT_ID]: reason});
```

Comment:
Note: We _should_ be able to use `Onyx.set` here, but it doesn't work because `Form` interferes with it. can't recall the details just now...

Reply:
Update: not sure if this was fixed in Onyx (which was updated on main since I originally wrote this code), or if I was just holding something wrong. But I was able to use `Onyx.set` without issues so I made that change, since it's generally better for performance when it works for the use-case.